### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.10 es 4.19 (13 novembre)

### DISCUSSION THREAD

**Discussion by erinrrr** [sapienzastudentsnetwork/calcolo-delle-probabilita#327] [Esercizi M-Z]
![image](https://github.com/user-attachments/assets/4da2f8bf-1cd3-499d-9bdd-e32b9b10febe)

**Comment by erinrrr** [comment]
![Screenshot_20241112_170738_Samsung Notes](https://github.com/user-attachments/assets/f9dd6cc4-6b07-421d-881f-7a6524c203ce)

**Comment by scuty2000** [comment]
ATTN! Duplicato di https://github.com/sapienzastudentsnetwork/calcolo-delle-probabilita/discussions/322
